mentoring/choosing_a_solution.md

```diff
@@ -11,8 +11,12 @@ The UI should look something like this:
 
 ![Mentoring queue](https://exercism-static.s3.eu-west-1.amazonaws.com/docs/mentor_queue.png)
 
-At the top of the queue panel, you will find a text box that allows you to filter by the student's name. To the right of this, you can the sort requests by oldest first, most recent first, student name or exercise name. On the right-hand side of the interface, you can filter by language or exercise name.
-You can also choose to only show exercises you've completed yourself (enabled by default), which is often wise as you may struggle to give great feedback if you've not wrestled with solving the exercise yourself. The list of exercises with outstanding mentorship-requests can be used to further filter the request list. Do this by selecting the exercise name (visible in the bottom right section of the screenshot above).
+At the top of the queue panel, you will find a text box that allows you to filter by the student's name.
+To the right of this, you can the sort requests by oldest first, most recent first, student name or exercise name.
+On the right-hand side of the interface, you can filter by language or exercise name.
+You can also choose to only show exercises that you have completed yourself (enabled by default), which is often wise as you may struggle to give great feedback if you have not wrestled with solving the exercise yourself.
```

Comment:
Is it enabled by default?  And does it matter?  If we change this, we would need to also remember to come back here to update this, and I am not sure that the reader would not notice that it is selected by default, if it is.

Reply:
I think it could be removed for reasons stated.  I agree with the line otherwise.  And I did not notice that it is the default behavior.  Even if it is, it removes maintenance cost by removing it now.